common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -148,6 +148,35 @@
     %>
 </%def>
 
+<%def name="renderReact(component, props=None)">
```

Comment:
This should invoke the `webpack()` def above -- I just copy-pasted cause I got sick of wading through the Mako docs.